### ISSUE TITLE
Rule: `keep-options-order`

### ISSUE THREAD

**Issue by Kelin2025** [effector/eslint-plugin#61]
### Story
For now, we can write `sample`/`guard` in any order:
```tsx
sample({
  target: foo, 
  fn: (bar) => Baz, 
  source: bar, 
  clock: clock
}) 
```
Would be nice to enforce a more convenient order
E.g. **source->clock->fn->target** (or clock first) 

### Extra stuff
- We also need the same thing for @sergeysova `patronum` I guess

**Comment by sergeysova** [comment]
Also, it should autofix

**Comment by doasync** [comment]
@igorkamyshev Why not use [the order](https://github.com/effector/effector/blob/42163eba87248812b56c01b2c5df8df5ecb36b30/packages/effector/index.d.ts#L929) of `fn` params?

What should be the order of `guard` options? 

> Linter should give your consistency, not options.
Linter can give options. You can use default ones for consistency. Or do you want to force your opinion without a choice?

**Comment by sergeysova** [comment]
I like the natural order of options: just like you read it.

`sample({ source, clock, fn, target })`
When **clock** is triggered, read data from **source**, pass it thru **fn**, and sent to **target**

As we see, order is clock, source, fn, target

---

The same for `guard`

`guard({ clock, source, filter, target })`

When **clock** is triggered, check **filter** for truthy, read data from **source**, send it to **target**.
Order is: clock, filter, source, target.

**Comment by doasync** [comment]
Yeah, I understand your point. It seems good as the default order. I could get used to that.

But you both ignore the fact that `source` data comes before `clock` data in the `fn` parameters. Many people are used to this alternative order. Why are you sure that the option is unacceptable here?
